accounter/accounter.go

```diff
@@ -248,14 +252,23 @@ func (a *Accounter) recvWork() {
 			reporter.GetInstance().IncAddressesFetched()
 
 			a.countMu.Lock()
-			a.checkedCount++
+			a.processedAddrCount++
 			a.countMu.Unlock()
 
 			a.addresses[resp.Address.Script()] = address{
 				path:     resp.Address,
 				txHashes: resp.TxHashes,
 			}
 
+			a.countMu.Lock()
+			for _, txHash := range resp.TxHashes {
+				if _, exists := a.transactions[txHash]; !exists {
+					a.backend.TxRequest(txHash)
+					a.seenTxCount++
```

Comment:
you want to increment `a.seenTxCount` before making the request?

Reply:
yes. sloppy coding 😅